Source/DafnyDriver/Commands/RunCommand.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Dafny;
+
+class RunCommand : ICommandSpec {
+  public string Name => "run";
+  public string Description => "Run the program.";
+
+  public IEnumerable<IOptionSpec> Options =>
+    new IOptionSpec[] {
+      TargetOption.Instance,
+      NoVerifyOption.Instance,
+    }.Concat(CommandRegistry.CommonOptions);
+
+  public void PostProcess(DafnyOptions dafnyOptions, Options options) {
+    dafnyOptions.EmitExecutable = true;
+    dafnyOptions.RunAfterCompile = true;
+    dafnyOptions.ForceCompile = NoVerifyOption.Instance.Get(options);
```

Comment:
To make `--noVerify` emulate the behavior of the /compile and /spillTargetCode options that would ignore the verification results. I'm not sure it's necessary to have this line since verification is skipped at the Boogie level, but for now I'd prefer sticking to the existing logic.